.jenkins/caffe2/test.sh

```diff
@@ -136,15 +136,15 @@ pip install --user pytest-sugar
 # torchvision tests #
 #####################
 if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
-  pip install -q --user git+https://github.com/pytorch/vision.git
+  pip install -q --user git+https://github.com/pytorch/vision.git@v0.5.0
   pip install -q --user ninja
   # JIT C++ extensions require ninja, so put it into PATH.
   export PATH="/var/lib/jenkins/.local/bin:$PATH"
   if [[ "$BUILD_ENVIRONMENT" == *py3* ]]; then
     # default pip version is too old(9.0.2), unable to support tag `manylinux2010`.
     # Fix the pip error: Couldn't find a version that satisfies the requirement
     sudo pip install --upgrade pip
-    pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==1.0.0.dev1123
+    pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==1.1.0.dev1228
```

Comment:
Could you update the onnx repo https://github.com/onnx/onnx/blob/49cc2f0/.circleci/build.sh#L46 as well? :-)

Reply:
Good catch! https://github.com/onnx/onnx/pull/2538

Reply:
Thanks!